### DISCUSSION TITLE
for help

### DISCUSSION THREAD

**Discussion by pizixi** [drakkan/sftpgo#424] [Q&A] [answered]
This is undoubtedly a great project, but it is not so friendly for beginners. I used the window version under the latest release for testing. The configuration file is also the default. The sqlite database. After starting the service, try to When adding a new user on webadmin, no matter what I enter in my home_dir, I get the same error, so I can’t perform the following test, but I don’t want to give up this matter, please help, thank you
![image](https://user-images.githubusercontent.com/56673215/118351915-27599600-b591-11eb-98ba-5c6ab8304ac7.png)

**Comment by drakkan** [comment, marked as answer]
Hi,

since you are on Windows you have to set the home dir to something like `C:\adir` (this is an absolute path on Windows)
